example/ZXingQt6CamReader.qml

```diff
@@ -9,7 +9,8 @@ import QtQuick.Controls
 import QtQuick.Layouts
 import QtQuick.Shapes
 import QtMultimedia
-import ZXing
+import ZXingQt
```

Comment:
What was the motivation behind this? From a QML users point of view, `ZXing` seemed to be the canonical name for that.

Reply:
You expose the Qt wrapper inside the ZXingQt namespace. I thought it would be more consistent if the QML wrapper is exposed with the same name. So, if someone uses your library from Qt, he should always use the name `ZXingQt` and never `ZXing` directly.

Another (consistent) option may be to drop the ZXingQt namespace and just use ZXing both in QML and as namespace in C++.

As said earlier, feel free to cherry pick from my proposed changes.

Reply:
- Combining QWidgets and QML in a single project is rather rare, but may be done: see [`QQuickWidget`](https://doc.qt.io/qt-6/qquickwidget.html).
 - The same developer may integrate ZXing in multiple applications, which may be a combination of QML and QWidgets based application depending on the needs of the customers of the developer.
 - In a single project, one may use ZXing from C++ and QML together.  Before this PR, that was even required as one had to call `ZXingQt::registerQmlAndMetaTypes()` and import `ZXing` in QML.

Nevertheless, if you prefer to keep using `ZXing` in QML, I can revert

Reply:
Yes, I'd prefer sticking with the `ZXing` name in QML.

Reply:
done